share/spice/geometry/geometry.js

```diff
@@ -56,8 +56,10 @@
             });
             if(svgNode.is(".fill"))
                 svgNode.css("fill", "transparent");
-            if(svgNode.is(".stroke"))
-                svgNode.css("stroke", "#000");
+            if(svgNode.is(".stroke")) {
```

Comment:
@moollaza Fixed. Thanks.

Instead of setting properties to original one, I just un-setted the properties so that they don't over-ride any CSS. :)